keepkeyctl

```diff
@@ -591,6 +595,10 @@ class Commands(object):
         (('-l', '--label'), {'type': str, 'default': ''}),
     )
 
+    test_recovery_sentence.arguments = (
+        (('-w', '--words'), {'type': int, 'choices': [12, 18, 24], 'default': 24}),
```

Comment:
IMO the default should be 12, since that's how the vast majority of KeepKey's in the wild are initialized.
```suggestion
        (('-w', '--words'), {'type': int, 'choices': [12, 18, 24], 'default': 12}),
```

Reply:
Yea I was wondering about that. It could really go either way in the sense that with a longer default, you can just hit "enter" after the last word and it will work (though the UI doesn't actually tell you this in the console while you are going through the workflow), but you are right that it should match what the majority will be using.

Reply:
Sounds like the firmware needs a "too few words entered" error to catch that case.